### ISSUE TITLE
Activate plugin tabindex through configuration (Tabindex omitted from control bar plugins)

### ISSUE THREAD

**Issue by karendolan** [polimediaupv/paella#484]
# Describe the bug
UPV Paella 5x assigned tabindex to plugins with getIndex() method so the tab key could be used to navigate them. In UPV Paella 6x (ala the Feb 4th commit), only plugins that have a getAriaLabel() method will have their getIndex() method accessed to set the tabindex on that pluginbutton. The volume plugin is the only plugin with that new method. Other plugins, like the flex skip forward, flex skip back, video qualities, fullscreen, etc, do not have the getAriaLabel() and the buttons are not given a tabIndex.
The playbackBar has a tabindex because it is manually assigned one in 06_ui_controls.js

# To Reproduce
1. Go to a Paella 6x video (https://paellaplayer.upv.es/paella/player-6.3.2/?id=belmar-multiresolution)
2. Click on tab key at least 4 times.
3. Notice that the tab index cycles between browser URL input box, player playback bar, and player volume button. But,  not to any of the other control bar controls.

# Workaround (if any)
Use the mouse to navigate control bar plugin controls.

# Environment Information
 - OS: MacOS
 - Browser: Safari, Chrome, FF
 - Paella version 6x

# Additional context
I'm adding a getAriaLabel()  method to our site's custom control bar plugins so they are assigned a tabindex by the plugin manager. Is there any reason why I would not want to do this?

**Comment by ferserc1** [comment]
This behavior is implemented that way on purpose. The idea is to improve usability for the visually impaired by hiding plugins that are not relevant to them.

We have not carried out usability tests in this sense, but until we have more information, the current operation seems to us to be much more adequate. However, we could study the possibility of making the ariaLabel parameter set in the configuration.

**Comment by karendolan** [comment]
Hi @ferserc1 enabling plugins to be tabindex accessible via configuration makes sense.  Then each site can decide what makes sense for their clients.

I suspect that tabindex enabled controls assists more people than vision impaired, as I've known paraplegics who heavily relied on keyboard navigation. I also started using the tab navigation, and noticed this change, when I got tired of using the mouse. However, I can see how too much tabindexing could annoy seeing impaired people.
